### PR TITLE
Feature/1151 new cielim related messages

### PR DESCRIPTION
**Pull request by AfrahGh**
* **Tickets addressed:** MAXGNC-1151
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description
Added two new cielim related messages, one containing rendering information, and another for celestial body parameters.

## Verification
No verification 

## Documentation
No documentation

## Future work
No future work

### CODE REVIEW

File: src/architecture/msgPayloadDefC/CameraRenderingMsgPayload.h
Comment (patkenneally) [inline src/architecture/msgPayloadDefC/CameraRenderingMsgPayload.h]: :small_red_triangle_down: This copyright statement is incorrect. It should read `2024, Laboratory for Atmospheric Space Physics, University of Colorado at Boulder`. This is the same for the other message added in this PR.

**Review by patkenneally — CHANGES_REQUESTED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

One change and then we can merge @AfrahGh

File: src/architecture/msgPayloadDefC/CameraRenderingMsgPayload.h
Comment (thibaudteil) [reply src/architecture/msgPayloadDefC/CameraRenderingMsgPayload.h]: done

**Review by patkenneally — APPROVED** [review]

**Pull request merged**
